com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 - Fixed: Hosting again after failing to host now works correctly
 
+- Fixed: NetworkManager properly cleaning up its clientlists upon shutdown (#1945)
+
```

Comment:
because "shutdown" is not exposed but StopXXX are
```suggestion
- Fixed NetworkManager to cleanup connected client lists after stopping (#1945)
```